Tests/SwiftFormatTests/Rules/AlwaysUseLowerCamelCaseTests.swift

```diff
@@ -210,4 +210,24 @@ final class AlwaysUseLowerCamelCaseTests: LintOrFormatRuleTestCase {
       ]
     )
   }
+
+  func testIgnoresFunctionsWithTestAttributes() {
+    assertLint(
+      AlwaysUseLowerCamelCase.self,
+      """
+      @Test
+      func function_With_Test_Attribute() {}
+      @Testing.Test("Description for test functions",
+            .tags(.testTag))
+      func function_With_Test_Attribute_And_Args() {}
+      func 1️⃣function_Without_Test_Attribute() {}
+      @objc
+      func 2️⃣function_With_Non_Test_Attribute() {}
+      """,
```

Comment:
Once you make the change above, you can add another example like `@Foo.Test` and verify that it does emit a diagnosis.

Reply:
Done.